src/dns/rfc3596.cc

```diff
@@ -25,7 +25,7 @@
 #include <strings.h>
 #endif
 
-#ifndef SQUID_RFC1035_H
+#ifndef RFC1035_MAXHOSTNAMESZ
```

Comment:
Since you are changing this manually, please update to match our current requirements for non-file wrappers:
```suggestion
#if !defined(RFC1035_MAXHOSTNAMESZ)
```